presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -283,4 +287,32 @@ public QueryManagerConfig setQueryExecutionPolicy(String queryExecutionPolicy)
         this.queryExecutionPolicy = queryExecutionPolicy;
         return this;
     }
+
+    @Min(1)
+    public int getInitialMinWorkersRequired()
+    {
+        return initialMinWorkersRequired;
+    }
+
+    @Config("query-manager.initial-min-workers-required")
+    @ConfigDescription("minimum number of workers required before queries are accepted")
+    public QueryManagerConfig setInitialMinWorkersRequired(int initialMinWorkersRequired)
+    {
+        this.initialMinWorkersRequired = initialMinWorkersRequired;
+        return this;
+    }
+
+    @NotNull
+    public Duration getInitializationTimeout()
+    {
+        return initializationTimeout;
+    }
+
+    @Config("query-manager.initialization-timeout")
+    @ConfigDescription("Maximum amount of wait time after which queries will be accepted regardless of minimum worker requirement")
```

Comment:
Maximum amount of time the cluster can remain in the `initializing` state (i.e. unable to accept queries)